### ISSUE TITLE
Nextcloud - Migration

### ISSUE THREAD

**Issue by easyfab** [truenas/apps#842]
I see this message : 
Warnings

    You are using a deprecated configuration, where both data and html nextcloud files are stored in the same path. Please consider updating your configuration. If you are unsure how to proceed, feel free to open a github issue.
    The backwards compatibility will be removed in the next major release.

For the moment I have this settings that is working  : 
![image](https://github.com/user-attachments/assets/8886282f-f097-424e-a100-ad07bf7087fc)

And the folders in data directory ( Nextcloud_data )
![image](https://github.com/user-attachments/assets/340942b9-ddfc-4f60-83cf-e87f33ddb9f8)

I don't understand what to move ?

Should I only move the data folder from Nextcloud_data tosomewhere else and change   Nextcloud User Data Storage path ?
I try this and unselect   "DEPRECATED: Old Storage Structure " but it doesn't work .

Could you help me to understand what to do .

**Comment by Spunky17** [comment]
I could be wrong but I believe that the directories in question that you have entered need to point to the specific directories in the list.  The entries you all have pointing to the top level directory and it needs to be moved one level deeper so that they are all not the same.

For example for AppData entry: /mnt/Data_pool/Nextcloud_data -> /mnt/Data_pool/Nextcloud_data/Appdata

**Comment by easyfab** [comment]
@Spunky17  Yes I did that but it doesn't work at first.

I kept  /mnt/Data_pool/Nextcloud_data/data unchanged and moved the others folders in /mnt/Data_pool/Nextcloud_data/Appdata

After reading the log, I see that I have a problem with a newly created /mnt/Data_pool/Nextcloud_data/Appdata/data  empty folder.
I delete this empty folder and try to re-run the app. It recreate this empty folder but now it runs ok and I have all my data .
Not obious why it works now and not before deleting this empty folder.
